src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -102,6 +135,21 @@ export function ConfigEditor(props: Props) {
     }
   };
 
+  // Workgroups
+  const [workgroupEndpoint, setWorkgroupEndpoint] = useState('');
+  const fetchWorkgroups = async () => {
+    try {
+      const res: Workgroup[] = await getBackendSrv().get(resourcesURL + '/workgroups');
+      return res.map((w) => ({
+        label: w.workgroupName,
+        value: w.workgroupName,
+        description: `${w.endpoint.address}:${w.endpoint.port}`,
+      }));
+    } catch {
```

Comment:
would it make sense to at least console.error an error if we got one rather than ignore it?

Reply:
Right, same thing applies to the existing code for provisioned Cluster. Will fix both. Will follow the style from opensearch-datasource. 
https://github.com/grafana/opensearch-datasource/blob/34a4fdd68a4bb022e1b8ef625ea639f1f0e65769/src/tracking.ts#L18-L19

Reply:
Updated at 649a3e623f7f52c9406147320fde974088390618.